tcl/cast.tcl

```diff
@@ -79,6 +79,14 @@ proc qc::cast_period {string} {
     return [qc::cast period $string]
 }
 
+proc qc::cast_url {string} {
+    return [qc::cast url $string]
+}
+
+proc qc::cast_relative_url {string} {
+    return [qc::cast relative_url $string]
+}
+
```

Comment:
`#|` comments.

If I understand correctly then these `qc::cast_` procs are required for `qc::checks` to function. If so then the `#|` should also denote these procs as deprecated since they will be superceded by the ensemble procs once current projects are fully migrated.